docs/index.html

```diff
@@ -20,8 +20,9 @@ <h3>Looking for the Hagglehaul Web App?</h3>
 <h3>Index</h3> 
 <ul>
     <li><a href="hagglehaul-coverage.html" target="_blank"> Test Coverage </a></li>
-    <li>API Endpoint Documentation</li> 
-    <li>Backend Documentation</li> 
+    <li>API Endpoint Documentation</li>
```

Comment:
API endpoint documentation is the swagger documentation. remove the link that say Swagger and link it to API Endpoint Documentation

Reply:
also, it's better convention to say 

```html
href="hagglehaul-swagger/"
```
instead of 

```html
href="hagglehaul-swagger/index.html"
```

(they mean the same thing)